assets/src/dashboard/app/reducer/templates.js

```diff
@@ -20,17 +20,23 @@
 import groupBy from '../../utils/groupBy';
 
 export const ACTION_TYPES = {
-  CREATE_TEMPLATE_FROM_STORY: 'create_template_from_story',
+  CREATING_TEMPLATE_FROM_STORY: 'create_template_from_story',
+  CREATE_TEMPLATE_FROM_STORY_FAILURE: 'create_template_from_story_failure',
+  CREATE_TEMPLATE_FROM_STORY_SUCCESS: 'create_template_from_story_success',
   LOADING_TEMPLATES: 'loading_templates',
   FETCH_TEMPLATES_SUCCESS: 'fetch_templates_success',
   FETCH_TEMPLATES_FAILURE: 'fetch_templates_failure',
+  FETCH_MY_TEMPLATES_SUCCESS: 'fetch_my_templates_success',
+  FETCH_MY_TEMPLATES_FAILURE: 'fetch_my_templates_failure',
```

Comment:
Separated this from generic 'fetch_my_templates' because it sounds like we're going to have several different types of templates to aggregate or flip through on the saved templates view and i think for now keeping these in their own lanes makes the most sense as new sections of data become available to us.